eZ/Publish/API/Repository/Tests/SearchServiceFulltextTest.php

```diff
@@ -194,21 +304,45 @@ public function testFulltextContentSearch($searchString, array $expectedKeys, ar
     }
 
     /**
-     * Test for the findLocations() method.
+     * Test for the findLocations() method on Solr 6.
      *
      * @param $searchString
      * @param array $expectedKeys
      * @param array $idMap
      *
      * @depends testPrepareContent
-     * @dataProvider providerForTestFulltextSearch
+     * @dataProvider providerForTestFulltextSearchSolr6
      */
-    public function testFulltextLocationSearch($searchString, array $expectedKeys, array $idMap)
+    public function testFulltextLocationSearchSolr6($searchString, array $expectedKeys, array $idMap): void
     {
-        if (($solrVersion = getenv('SOLR_VERSION')) && $solrVersion < 6) {
-            $this->markTestSkipped('Solr 4 detected, skipping as scoring won\'t match');
+        if (($solrVersion = getenv('SOLR_VERSION')) && ($solrVersion < 6 || $solrVersion >= 7)) {
```

Comment:
So after removal of a lot of cluttering code now I see clearly this comparison and I'm not happy about it. We should parse version string and check sort of "not equals" condition. Can the [`\version_compare`](https://www.php.net/manual/en/function.version-compare.php) function help us in that regard somehow? If not, then maybe just a simple regex?